### ISSUE TITLE
Accessibility: Object Status: All object statuses are read continuously at first tab stop when accessed in virtual pc mode

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3446]
Description: All object statuses are read continuously at first tab stop when accessed in virtual pc mode

Expected: All object statuses should not be read continuously. different object statuses should be read separately

Screenshot: 
![image](https://user-images.githubusercontent.com/32538291/94520906-bf954b80-024a-11eb-9097-55c2c7f2ab8b.png)

![image](https://user-images.githubusercontent.com/32538291/94520915-c2903c00-024a-11eb-82da-c5dde13382a4.png)

**Comment by Lokanathan-k** [comment]
fixed as part of the PR https://github.com/SAP/fundamental-ngx/pull/4176
